cherwell_connector.py

```diff
@@ -61,16 +61,15 @@ def _process_html_response(self, response, action_result):
         try:
             soup = BeautifulSoup(response.text, "html.parser")
```

Comment:
Add the below script to remove unnecessary things from HTML response

# Remove the script, style, footer and navigation part from the HTML message
	for element in soup(["script", "style", "footer", "nav"]):
		element.extract()